### PR TITLE
[WIP] Add CLI option to select custom targets path

### PR DESCRIPTION
**Pull request by supra08**
Fixes issue #275 

## Proposed Changes

  - Add CLI flag to add custom targets path
  - Expand the path and save it as a reclass object

There is only one problem that I am facing:
`kapitan/lib/kapitan.libjsonnet` has this
```
  inventory(target=std.extVar("target"), inv_path="inventory/"):: std.native("inventory")(target, inv_path),
```
hardcoded. So is there a way to override this when dealing with custom target paths?
@uberspot

### CODE REVIEW

**Comment by supra08** [comment]
@googlebot I signed it!

File: kapitan/resources.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ def inventory(search_paths, target, inventory_path="inventory/"):
     return inventory_reclass(full_inv_path)["nodes"][target]
 
 
-def inventory_reclass(inventory_path):
+def inventory_reclass(inventory_path, targets_path=None):
     """
     Runs a reclass inventory in inventory_path
```
Comment (uberspot) [inline kapitan/resources.py]: Can you update this docstring as well to reflect the new behavior?

File: kapitan/targets.py [hunk 2]
```diff
@@ -563,8 +573,11 @@ def schema_validate_compiled(targets, compiled_path, inventory_path, schema_cach
     worker = partial(schema_validate_kubernetes_output, cache_dir=schema_cache_path)
     pool = multiprocessing.Pool(parallel)
 
+    if targets_path == None:
+        targets_path = inventory_path
```
Comment (uberspot) [inline kapitan/targets.py]: Shouldn't this be set to `os.path.join(inventory_path, "targets")` ? 
Or am I missing something?

Comment (supra08) [reply kapitan/targets.py]: Yeah I missed out here, will fix it.

File: kapitan/resources.py (hunk 1 above)
Comment (supra08) [reply kapitan/resources.py]: alright!

**Comment by uberspot** [comment]
Besides my comment, the only other thing missing is a small test to check that this flag is actually used. Could you add one under `kapitan/tests/test_compile.py` ? :)

**Comment by supra08** [comment]
Yeah sure! I'll add it ASAP. Apologies for the delay.

**Pull request closed without merging**